internal/pkg/aws/elbv2/elbv2.go

```diff
@@ -153,3 +156,62 @@ func (t *TargetHealth) HealthStatus() *HealthStatus {
 func (t *TargetHealth) targetID() string {
 	return aws.StringValue(t.Target.Id)
 }
+
+// LoadBalancer contains information about a given load balancer.
+type LoadBalancer struct {
+	ARN            string
+	Name           string
+	DNSName        string
+	Scheme         string // "internet-facing" or "internal"
+	SecurityGroups []string
+}
+
+// LoadBalancer returns select information about a load balancer.
+func (e *ELBV2) LoadBalancer(id string) (*LoadBalancer, error) {
+	var input *elbv2.DescribeLoadBalancersInput
+	if arn.IsARN(id) {
+		input = &elbv2.DescribeLoadBalancersInput{
+			LoadBalancerArns: []*string{aws.String(id)},
+		}
+	} else {
+		input = &elbv2.DescribeLoadBalancersInput{
+			Names: []*string{aws.String(id)},
+		}
+	}
+	output, err := e.client.DescribeLoadBalancers(input)
+	if err != nil {
+		return nil, fmt.Errorf("describe load balancer %q: %w", id, err)
+	}
+	lb := output.LoadBalancers[0]
```

Comment:
⚠️ index out of range. Ideally this shouldn't happen because the SDK should return an error but just in case